rust/rbac-registration/src/cardano/cip509/utils/cip134_uri_set.rs

```diff
@@ -142,7 +155,7 @@ impl Cip0134UriSet {
     /// 2: [uri_4]
     /// ```
     #[must_use]
-    pub fn update(
+    pub(crate) fn update(
```

Comment:
Could you consider adding a UT for this function to verify if it properly manages the removal and insertions from `x_uris`, `c_uris`, `taken_stake_addresses`, etc.?

Reply:
yes, but later on